build_library/release_util.sh

```diff
@@ -113,6 +115,13 @@ compress_disk_images() {
                     processed_format["${format}"]=1
                 fi
             done
+            # If requested, delete the input file after compression (only if 'none' is not part of the formats)
```

Comment:
Could you expand on why two names are excluded here?

Reply:
Yes, good idea